packages/samples/vue-webview-sample/package.json

```diff
@@ -21,8 +21,8 @@
     ]
   },
   "scripts": {
-    "vscode:prepublish": "yarn build",
-    "build": "npx tsc -p ./ && cd webviews/vue-sample && yarn && yarn build",
```

Comment:
We'll still need `cd webviews/vue-sample && ...` as part of the `build` script or otherwise it will not build the `dist` folder for the webview.

Reply:
if you check `pnpm-workspace.yaml` we add the relevant workspaces for this exact command, which should bundle these packages without the need of manually adding these, you can test this by running the debugger and seeing that the webview works as intended.

Reply:
Ah, I stand corrected. I didn't notice that new `dist` folders were generated for the webviews when I last tested - but after testing again, the webviews are compiled as expected. Sorry for the confusion 😓